Makefile

```diff
@@ -88,6 +88,10 @@ GAIA_DEV_PIXELS_PER_PX?=1
 NGA_SERVICE_WORKERS?=0
 FIREFOX_SYNC?=0
 
+ifeq ($(GAIA_DEVICE_TYPE),tv)
+  FIREFOX_SYNC=1
+endif
+
```

Comment:
Great idea!
How about name it as ```device.mk```?
So any devices could place the same file for its build/config folder.
e.g.
```
build/config/tv/device.mk
build/config/phone/device.mk
build/config/tablet/device.mk
```